src/editor/Core/Editor.re

```diff
@@ -36,7 +36,7 @@ type viewport = {
   pixelHeight: int,
 };
 
-let getVisibleLines = (view: t) => view.size.pixelWidth / view.lineHeight;
+let getVisibleView = (view: t) => view.size.pixelHeight / view.lineHeight;
```

Comment:
Fixed a bug here, but also thought that 'view' is a more appropriate term here.

Its not really the visible lines, since if I only have 5 lines on screen, this still says 50 since I can see 50 at one time. I should probably add a helper back here to get the number of visible lines as well.

Reply:
Makes sense! Thanks for improving this 👍